bsp/cvitek/drivers/drv_hw_i2c.h

```diff
@@ -0,0 +1,233 @@
+/*
+ * Copyright (c) 2006-2023, RT-Thread Development Team
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * Change Logs:
+ * Date           Author       Notes
+ *2024-02-14      ShichengChu  first version
+ */
+#ifndef __DRV_HW_I2C_H__
+#define __DRV_HW_I2C_H__
+
+#include <rtthread.h>
+#include "rtdevice.h"
```

Comment:
```suggestion
#include <rtdevice.h>
```
只需要rtdevice.h即可 里边包含了rtthread.h